planning/static_centerline_generator/src/static_centerline_generator_node.cpp

```diff
@@ -431,14 +437,14 @@ std::vector<lanelet::Id> StaticCenterlineGeneratorNode::plan_route(
     mission_planner->initialize(&node, map_bin_ptr_);
 
     // plan route
-    const auto route = mission_planner->plan(check_points);
+    auto route = mission_planner->plan(check_points);
 
     return route;
   }();
   RCLCPP_INFO(get_logger(), "Planned route.");
 
   // get lanelets
-  const auto route_lane_ids = get_lane_ids_from_route(route);
+  auto route_lane_ids = get_lane_ids_from_route(route);
```

Comment:
Why did you remove const?

Reply:
It was a clang-tidy warning

Reply:
Do you know why the clang-tidy recommended it? Removing const does not make sense to me.